docs/manual/forms/basics.rst

```diff
@@ -78,6 +78,13 @@ In dit tabblad worden de algemene formulier gegevens weergegeven.
   bijvoorbeeld alleen informatie of verwijzingen naar andere formulieren. In dat
   geval vinkt u deze optie uit.
 
+.. note::
+
+  Als de formuliernaam lange woorden bevat, is het mogelijk om `soft hyphens`_ (unicode U+00AD) te gebruiken om door
+  te geven waar het woord kan worden afgebroken om naar de volgende regel te gaan.
```

Comment:
Keep in mind that the target audience for the manual are non-technical people, the term "soft hyphen" is probably too complicated and unicode definitely is.

Instead, you should probable describe this that an "optioneel koppelteken kan ingevoerd worden" where they want the word to break, and then some instructions on how exactly to do that. I think one suggestion was the <kbd>ctrl</kbd> + <kbd>-</kbd> combination in Word and then copy-pasting that, but we need to verify that actually works.